lib/arrays.md

```diff
@@ -5,44 +5,53 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+An array basically acts as a list of variables that can be modified, ordered, recalled, etc. however you command later on.
```

Comment:
Here is an example array:

```ruby
["Sumeet", 50, "mel-d", "Orange"]
```

Your definition is that an array "acts as a list of variables". But there are no _variables_ in my array above.

Can you clarify your definition?